mail_composer_cc_bcc/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Email CC and BCC",
+    "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
+    "version": "15.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Social",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "maintainers": ["hailangvn2023"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "preloadable": True,
+    "depends": [
+        "mail",
+        "sale_management",
```

Comment:
```suggestion
```

Do this module need depend module `sale_management` ?

Reply:
You are right. The PR has been updated.

Reply:
Thanks for your review.